crypto/fipsmodule/kdf/sskdf.c

```diff
@@ -174,6 +175,9 @@ static int sskdf_variant_hmac_compute(sskdf_variant_ctx *ctx, uint8_t *out,
 
   uint32_t written;
 
+  // NIST.SP.800-56Cr2: Step 6.2 HMAC-hash(salt, counter || secret || info)
+  // Note: |variant_ctx->hmac_ctx| is already initalized with the salt during
+  // it's initial construction.
```

Comment:
Nit:
```Suggestion
  // its initial construction.
```

Reply:
Will make this documentation correction in the KDF counter PR.